pkg/opentelemetry/collector/client/config.go

```diff
@@ -0,0 +1,33 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/plgd-dev/hub/v2/pkg/net/grpc/client"
+)
+
+type GRPCConfig struct {
+	Enabled    bool          `yaml:"enabled"`
+	Connection client.Config `yaml:",inline"`
+}
+
+func (c *GRPCConfig) Validate() error {
+	if !c.Enabled {
+		return nil
+	}
+	if err := c.Connection.Validate(); err != nil {
```

Comment:
Lines 18-21 can be shortened to:

`
c.Connection.Validate()
`

Reply:
Fixed